Week2/10_program.js

```diff
@@ -1,4 +1,17 @@
-let myArray =["My age","30"]; // With array is possibale to store different data, within a single variable
+// 10.1 Can you store multiple types in an array? Numbers and strings? Make an example that illustrates your answer.
+
+let myArray =["My age",30 , ["Student" ,5]]; // With array is possibale to store different data, within a single variable
 console.log(myArray);
-let maxNumber = Math.pow(10, 1000);
-console.log(maxNumber);
\ No newline at end of file
+
+
+//10.2 Can you compare infinities? (Not in Eyad's world) - does 6/0 === 10/0? How can you test this?
+//10.3 Add console.log statements to the above program
+let inf1 = 10/0;
+let inf2 = 6/0;
+
+function comparInfinities(inf1,inf2){
+        if (inf2 = Number.MAX_VALUE , inf2 = Number.MAX_VALUE)
+        console.log("inf1 and inf2 are infinity")
+        else console.log("False")
+}
+comparInfinities()
```

Comment:
You call the function, which requires 2 `arguments` and then pass no `arguments`, so this will not work. But, you don't have to use a function, so I would try to avoid using a function because it might be confusing right now.